docs/cli/graphback-cli.md

```diff
@@ -2,4 +2,80 @@
 id: graphback-cli
 title: Graphback CLI
 sidebar_label: Graphback CLI
----
\ No newline at end of file
+---
+
+Graphback helps you to kickstart your experience with any existing GraphQL implementation by generating backend and client side CRUD layer using your [GraphQL data model](../model/datamodel.md). The CLI exposes a single `generate` command.  
+
+## Installation
+
+You can add `Graphback CLI` on your existing GraphQL project using either of the following commands. 
+
+With npm:
+
+```bash
+npm install --save-dev graphback-cli
+```
+
+Or with yarn: 
+
+```bash
+yarn add --dev graphback-cli
+```
+
+## Usage
+
+### Pre-requisite
+
+You project must contain the configuration file that follows the format described by the [graphql-config](https://graphql-config.com/introduction).
+
+Taking a `.graphqlrc.yml` configuration file for example, your configuration should have the following minimum required entry:
+
+```yaml
+## Please review configuration and adjust it for your own project
+schema: ./src/schema/schema.graphql
+extensions:
+  graphback:
+    model: ./model/**/*.graphql
+    ## You can add as many Graphback plugins as you wish
+    ## Here we are only adding the Schema and Client plugin
+    plugins:
+      graphback-schema:
+        format: graphql
+        outputPath: ./src/schema/schema.graphql
+      graphback-client:
+        format: 'graphql'
+        outputFile: ./src/graphql/graphback.graphql   
+```
+
+:::note
+If you used [graphql-cli](https://github.com/Urigo/graphql-cli) to initialise the project, this file will be automatic setup for you. Otherwise, you can still use the `init` command from the cli to initialise this configuration setup.
```

Comment:
I know that UK English uses `s` instead of `z`, but since graphback.dev targets the world, I wonder which we should use? https://stackoverflow.com/a/157837

```suggestion
If you used [graphql-cli](https://github.com/Urigo/graphql-cli) to initialise the project, this file will be automatic setup for you. Otherwise, you can still use the `init` command from the CLI to initialize this configuration setup.
```